cctp_primitives/src/bit_vector/compression.rs

```diff
@@ -241,13 +279,12 @@ mod test {
 
     #[test]
     fn expected_bit_vector_size() {
-        let seed: u64 = rand::thread_rng().gen();
-        let original_bit_vector: Vec<u8> = generate_random_bit_vector(seed);
-
-        let mut wrong_bit_vector_size: usize = original_bit_vector.len();
+        let mut seed: u64 = 0;
+        let mut original_bit_vector: Vec<u8> = Vec::new();
 
-        while wrong_bit_vector_size != original_bit_vector.len() {
-            wrong_bit_vector_size = rand::thread_rng().gen();
+        while original_bit_vector.len() <= 0 || original_bit_vector.len() > MAX_DECOMPRESSION_SIZE {
```

Comment:
considering the comment above, we may just use `let original_bit_vector: Vec<u8> = generate_random_bit_vector(seed);` once as before.

Reply:
Fixed.